packages/@aws-cdk/region-info/build-tools/fact-tables.ts

```diff
@@ -500,3 +500,218 @@ export const FIREHOSE_CIDR_BLOCKS: { [region: string]: string } = {
   'us-west-1': '13.57.135.192',
   'us-west-2': '52.89.255.224',
 };
+
+const ADOT_LAMBDA_LAYER_JAVA_SDK_ARNS: { [version: string]: { [arch: string]: { [region: string]: string } } } = {
```

Comment:
Do we need to hard code the account id multiple times? Can't we generate all the permutations of region/accountid/language programmatically?

I know that you are following what was in the repo before, but I'm curious if we cannot consider this. It is very easy for a typo to happen here.